app/static/src/app/components/run/RunModelPlot.vue

```diff
@@ -5,7 +5,7 @@
 
 <script lang="ts">
 import {
-    computed, defineComponent, ref, watch
+    computed, defineComponent, ref, watch, onMounted, onUnmounted
 } from "vue";
 import { useStore } from "vuex";
 import { EventEmitter } from "events";
```

Comment:
Why? My previous comment about mounting and unmounting still stands even with the two watches combined:
> on Mounted now needed to redraw the model when switch tab back from another, and onUnmounted to disconnect the resizeObserver when the user closes the tab.

Reply:
Arguably could do a `drawPlot` rather than `runModel` in the onMounted - but this avoid possible race condition of user switching rapidly to other tab before initial run model was fired after odin code fetched.

Reply:
I actually commented on line 11. You probably thought line 8. Sorry for the confusion.